lib/Home/screens.dart

```diff
@@ -93,8 +93,24 @@ class GifCard extends StatelessWidget {
   Widget build(BuildContext context) {
     final mediaquery = MediaQuery.of(context);
 
-    return Container(
-      color: Theme.of(context).colorScheme.background,
+    return Material(elevation: 0,child: Container(
+
```

Comment:
Hi,
Why have you used Material here?
You could've used the `Card` widget instead.

Reply:
Actually initially I was proiding the elevation through the material widget, but it was not possible to provide a box-elevation there. So, i basically did this via container decoration itself, and did the material elevation to 0. Nevertheless, I've removed the material widget @bhattabhi013 so can u please review it now?